_chapters/chapter_14.md

```diff
@@ -303,7 +317,9 @@ layout: rails_tutorial
 
       I can think of one - logged out users shouldn't be able to add books. This is *your* super rad bookstore after all. You don't want some random person to come along and start adding books to your bookstore.
 
-  1.  Open `app/views/books/index.html.erb` and find the "Add a book" link. Update the template so the "Add a book" link is only visible to logged in users. Logged out users shouldn't see anything.
+  1.  Open `app/views/books/index.html.erb` and find the "Add a book" link. 
+
+      Take some time and try updating the template so the "Add a book" link is only visible to logged in users. Logged out users shouldn't see anything.
```

Comment:
A couple of thoughts...
  * Should this sentence be a separate step?
  * I like "Update the template..," more than "Take some time and try updating..." because it *sounds* more confident/affirmative, if that make sense. This is getting closer to the end of tutorial, and I like having the more confident language. Then again, I could see it being too cold.